cmd/resize.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+    "errors"
+
+    "github.com/getcarina/carina/console"
+    "github.com/spf13/cobra"
+)
+
+func newResizeCommand() *cobra.Command {
+    var options struct {
+        name     string
+        nodes    int
+        wait     bool
+    }
+
+    var cmd = &cobra.Command{
+        Use:               "resize <cluster-name>",
+        Short:             "Resize a cluster",
+        Long:              "Resize a cluster",
```

Comment:
For the long description, maybe let's make it a bit more clear what "resize" means?

For example:
`Resize a cluster by setting the number of worker nodes`